addons/base_import/tests/test_csv_magic.py

```diff
@@ -150,3 +150,47 @@ def test_parse_float(self):
             ['1.62', '-1.62', '+1.62', '+1.62', '-1.62',
              '1234567.89', '1234567.89']
         )
+
+
+class TestLanguageDetection(ImportCase):
+    def test_preview_language_detection(self):
```

Comment:
Linting: don't forget to add a void line above

Reply:
Yes thanks, changed.